js/src/ui/Actionbar/Search/search.js

```diff
@@ -125,54 +125,56 @@ export default class ActionbarSearch extends Component {
   handleTokenAdd = (value) => {
     const { tokens } = this.props;
 
-    const newSearchValues = uniq([].concat(tokens, value));
+    const newSearchTokens = uniq([].concat(tokens, value));
```

Comment:
That's just in case tokens isn't really an `Array`, but I guess that's taken care of by the `PropTypes` Object